documentation/modules/ref-brokered-infra-config-fields.adoc

```diff
@@ -10,24 +10,28 @@
-|broker.podTemplate.spec.tolerations |Specifies the toleration settings for the broker pod, which allows this pod to run on certain nodes that other pods cannot run on.
-|broker.storageClassName |Specifies what storage class to use for the persistent volume for the broker.
-|broker.updatePersistentVolumeClaim |If the persistent volume supports resizing, setting this value to `true` allows the broker storage to be resized.
+|`version` |Specifies the {ProductName} version used. When upgrading, {ProductName} uses this field to determine whether to upgrade the infrastructure to the requested version.
+|`admin.resources.memory` |Specifies the amount of memory allocated to the admin Pod.
+|`admin.podTemplate.metadata.labels` |Specifies the labels added to the admin Pod.
+|`admin.podTemplate.spec.priorityClassName` |Specifies the priority class to use for the admin Pod so you can prioritize admin Pods over other Pods in the {KubePlatform} cluster.
+|`admin.podTemplate.spec.affinity` |Specifies the affinity settings for the admin Pod so you can specify where on particular nodes a Pod runs, or if it cannot run together with other instances.
+|`admin.podTemplate.spec.tolerations` |Specifies the toleration settings for the admin Pod, which allows this Pod to run on certain nodes that other Pods cannot run on.
+|`broker.addressFullPolicy` |Specifies action taken when a queue is full: `BLOCK`, `FAIL`, `PAGE`, `DROP`. The default value is `PAGE`. For more information see the link:{LinkaddressFullPolicyDocs}[{BrokerName} documentation^].
+|`broker.globalMaxSize` |Specifies the maximum amount of memory used for queues in the broker.
+|`broker.resources.memory` |Specifies the amount of memory allocated to the broker.
+|`broker.resources.storage` |Specifies the amount of storage requested for the broker.
+|`broker.podTemplate.metadata.labels` |Specifies the labels added to the broker Pod.
+|`broker.podTemplate.spec.priorityClassName` |Specifies the priority class to use for the broker Pod so you can prioritize broker Pods over other Pods in the {KubePlatform} cluster.
+|`broker.podTemplate.spec.affinity` |Specifies the affinity settings for the broker Pod so you can specify where on particular nodes a Pod runs, or if it cannot run together with other instances.
+|`broker.podTemplate.spec.tolerations` |Specifies the toleration settings for the broker Pod, which allows this Pod to run on certain nodes that other Pods cannot run on.
+|`broker.podTemplate.spec.containers.initialDelaySeconds` |Specifies probe delay value in seconds for the broker Pod.
+|`broker.podTemplate.spec.containers.timeoutSeconds` |Specifies probe timeout value in seconds for the broker Pod.
+|`broker.podTemplate.spec.containers.resources` |Specifies broker Pod resource requests and limits for CPU and memory.
+|`broker.podTemplate.spec.containers.env` |Specifies environment variables for the broker Pod.
```

Comment:
@k-wall, Can you confirm this description is correct?

Reply:
Yes, definition is good.